manifests/rake.pp

```diff
@@ -22,7 +22,7 @@
     command     => "/usr/sbin/foreman-rake ${title}",
     user        => $user,
     environment => sort(join_keys_to_values({ 'HOME' => $app_root } + $environment, '=')),
-    logoutput   => 'on_failure',
```

Comment:
This was set to `on_failure` (which is default, btw) since the original implementation in 88e4c16403df1c55376de7ff7dd83420a7a000af